clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -43,10 +43,10 @@
     public static final String METADATA_MAX_AGE_DOC = "The period of time in milliseconds after which we force a refresh of metadata even if we haven't seen any partition leadership changes to proactively discover any new brokers or partitions.";
 
     public static final String SEND_BUFFER_CONFIG = "send.buffer.bytes";
-    public static final String SEND_BUFFER_DOC = "The size of the TCP send buffer (SO_SNDBUF) to use when sending data.";
+    public static final String SEND_BUFFER_DOC = "The size of the TCP send buffer (SO_SNDBUF) to use when sending data.If the value is -1, the OS default will be used.";
```

Comment:
A space is missing after the period. Seems like all the other comment updates have the same issue.